core/controllers/learner_group.py

```diff
@@ -790,6 +860,67 @@ def put(self, learner_group_id):
         })
 
 
+class ExitLearnerGroupHandler(base.BaseHandler):
+    """Handles a learner exiting from a learner group."""
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    URL_PATH_ARGS_SCHEMAS = {
+        'learner_group_id': {
+            'schema': {
+                'type': 'basestring',
+                'validators': [{
+                    'id': 'is_regex_matched',
+                    'regex_pattern': constants.LEARNER_GROUP_ID_REGEX
+                }]
+            },
+            'default_value': None
+        }
+    }
+
+    HANDLER_ARGS_SCHEMAS = {
+        'PUT': {
+            'learner_username': {
+                'schema': {
+                    'type': 'basestring',
+                    'validators': [{
+                        'id': 'has_length_at_most',
+                        'max_value': constants.MAX_USERNAME_LENGTH
+                    }]
+                }
+            }
+        }
+    }
+
+    @acl_decorators.can_access_learner_groups
+    def put(self, learner_group_id):
```

Comment:
Missing python typing for learner_group_id arg.

Reply:
If you are accepting the default value as None then please add Optional[str] but if only string values are expected then add str.